build.sbt

```diff
@@ -121,7 +114,6 @@ lazy val root = (project in file("."))
     Compile / packageDoc / publishArtifact := false,
     Debian / topLevelDirectory := Some(normalizedName.value),
     Debian / serverLoading := Some(Systemd),
-    debianPackageDependencies := Seq("java8-runtime-headless"),
```

Comment:
This was removed in the previous attempt, but I'm not sure why it's no longer required or doesn't need upgrading.

Reply:
@adamnfish has suggested that it "predates our base images ... it would have been used on ubuntu images that installed everything at start time"